acapy/start.sh

```diff
@@ -2,23 +2,33 @@
 
 set -euxo pipefail
 
-if [[ -z "$MEDIATOR_URL" ]]; then
-    MEDIATOR_URL=$(curl -s http://ngrok:4040/api/tunnels | jq -r '[.tunnels[] | select(.proto == "https")][0].public_url')
-fi
+MEDIATOR_URL=$(curl -s http://ngrok:4040/api/tunnels | jq -r '[.tunnels[] | select(.proto == "https")][0].public_url')
```

Comment:
Why was the check for the environment variable removed? Removing it will prevent this from working if run without ngrok (i.e.: running locally behind a reverse proxy to expose to the web).

Reply:
This was a mistake. I didn't notice. Thanks for catching it.